CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.5)
 project(mppic)
 
-set(XTENSOR_USE_XSIMD 1)
-set(XTENSOR_USE_OPENMP 1)
+add_definitions(-DXTENSOR_ENABLE_XSIMD)
+add_definitions(-DXTENSOR_USE_XSIMD)
```

Comment:
What was the purpose of this change ?

Reply:
Honestly, mostly paranoia. I couldn't see any changes with or without `XTENSOR_USE_OPENMP` / `XTENSOR_USE_XSIMD`. I don't know if (and certainly it shouldn't be the case that) calling it as a definition would help, but this is how I see it done in the guides for xtensor when looking into optimization options for compilation. So I didn't see a reason to argue with what others are doing.